src/translations/en.json

```
@@ -12,7 +12,9 @@
     "cancel": "Cancel",
     "ranked": "Ranked",
     "casual": "Casual",
-    "your": "your"
+    "your": "your",
+    "fullScreen": "full screen",
+    "exitFullScreen": "Exit Full Screen"
```

For the fullscreen pr
```suggestion
    "your": "your"
```